bower.json

```diff
@@ -25,7 +25,8 @@
     "jquery-selectric": "~1.8.7",
     "sly": "~1.2.7",
     "pym.js": "~0.4.1",
-    "beat-blocks": "~0.1.3"
+    "beat-blocks": "~0.1.3",
+    "numeral": "~1.5.3"
```

Comment:
Added [numeral library](http://numeraljs.com/) to handle general number formatting needs. This PR hardcodes it's use pretty specifically but in the future it will provide a good variety of options.